contrib/containers/ci/Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV DEBIAN_FRONTEND="noninteractive" TZ="Europe/London"
 # (zlib1g-dev is needed for the Qt host binary builds, but should not be used by target binaries)
 ENV APT_ARGS="-y --no-install-recommends --no-upgrade"
 
-
+SHELL ["/bin/bash", "-c"]
```

Comment:
it's not good to depends on `bash`, `sh` should be enough:

```
$ sh -c 'for i in a b c ; do echo $i ; done'
a
b
c
```

Reply:
`bash` is part of the base Ubuntu distribution and will be required in [dash#6387](https://github.com/dashpay/dash/pull/6387) regardless.